library/core/src/lib.rs

```diff
@@ -55,7 +55,28 @@
     html_playground_url = "https://play.rust-lang.org/",
     issue_tracker_base_url = "https://github.com/rust-lang/rust/issues/",
     test(no_crate_inject, attr(deny(warnings))),
-    test(attr(allow(dead_code, deprecated, unused_variables, unused_mut)))
+    test(attr(allow(dead_code, deprecated, unused_variables, unused_mut))),
+    cfg_hide(
+        not(test),
+        target_pointer_width = "16",
+        target_pointer_width = "32",
+        target_pointer_width = "64",
+        target_has_atomic = "8",
+        target_has_atomic = "16",
+        target_has_atomic = "32",
+        target_has_atomic = "64",
+        target_has_atomic = "ptr",
+        target_has_atomic_equal_alignment = "8",
+        target_has_atomic_equal_alignment = "16",
+        target_has_atomic_equal_alignment = "32",
+        target_has_atomic_equal_alignment = "64",
+        target_has_atomic_equal_alignment = "ptr",
+        target_has_atomic_load_store = "8",
+        target_has_atomic_load_store = "16",
+        target_has_atomic_load_store = "32",
+        target_has_atomic_load_store = "64",
+        target_has_atomic_load_store = "ptr",
```

Comment:
Why hide these?

Reply:
These, along with `target_has_atomic_ptr`, are unstable `cfg`. While I think it would be really nice to have this all documented on the items, I don't think the `doc(cfg)` presentation of them is good enough. I'll attach a screenshot of what it looks like with them unhidden.

I also think it might make sense to enhance `doc(cfg_hide)` to support specifying just the key of key-value cfg expressions so that these aren't duplicated for each size.